DataSetExplorer/DataSetBuilder/Model/DataSet.cs

```diff
@@ -8,11 +7,13 @@ public class DataSet
         public int Id { get; private set; }
         public string Name { get; private set; }
         public HashSet<DataSetProject> Projects { get; private set; }
-        
-        public DataSet(string name)
+        public List<CodeSmell> CodeSmells { get; private set; }
```

Comment:
Maybe a more descriptive name? What does this list of CodeSmells mean for the DataSet? `SupportedSmells`? `ExaminedSmells`?